### DISCUSSION TITLE
issue: useWatch does not trigger validation when input registered with disabled: false

### DISCUSSION THREAD

**Discussion by reinertisa** [react-hook-form/react-hook-form#11737] [Q&A]
### Version Number

7.51.1

### Codesandbox/Expo snack

https://codesandbox.io/p/sandbox/usewatch-does-not-trigger-validation-r2f55d?file=%2Fsrc%2FApp.js%3A52%2C30

### Steps to reproduce

1. Go to 'https://codesandbox.io/p/sandbox/usewatch-does-not-trigger-validation-r2f55d?file=%2Fsrc%2FApp.js%3A52%2C30'
2. Click on 'Submit'
3. See validation errors
4. Select 'Apple'
5. You will see validation error because of input registered as disabled: false - FAILED
6. Select 'Red'
7. You will see validation error will gone because input registered without disabled and validation will trigger. - PASSED

### Expected behaviour

When input registered with disabled is false, validation should trigger when I toggle input. It works with version 7.47.0 but it does not work with 7.51.1
**Version 7.51.1 - FAILED**
<img width="1029" alt="Screenshot 2024-03-25 at 12 27 56 PM" src="https://github.com/react-hook-form/react-hook-form/assets/71358643/a7c97537-2d6f-445f-ad8e-2a60da04c0bf">
**Version 7.47.0 PASSED**
<img width="1058" alt="Screenshot 2024-03-25 at 12 27 24 PM" src="https://github.com/react-hook-form/react-hook-form/assets/71358643/7fcf8552-5ad2-4dcd-8acb-1bd29fcdc545">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
use `validate` function instead.

**Comment by reinertisa** [comment]
@bluebill1049 I tried validate function and it does not work. If you want, I can create code sandbox for it.
